wiki/osu!_Program_Files/en.md

```diff
@@ -1,113 +1,115 @@
 
-For further reference, please refer to the [Skinning](/wiki/Skinning) page. Also, "osu! by peppy" is the only skin without its folder and cannot be deleted directly.
+*For further reference, please refer to the [Skinning](/wiki/Skinning) page.* 
 
 ### Songs
 
-**Your compendium of osu! beatmaps**. Usually contains .osu (difficulties), .mp3/.ogg (music file), .jpg/.png/.gif (BG image), .osb (SB file) and .mp4/.flv (video file). May also contains .wav/.ogg (hitsound file) and folders (SB spites and/or skin).
+**The Songs folder holds the player's osu! beatmaps**. Usually contains `.osu` (difficulties), `.mp3`/`.ogg` (music files), `.jpg`/`.png`/`.gif` (background images), `.osb` (storyboard files) and `.mp4`/`.flv` (video files). May also contain `.wav`/`.ogg` (hitsound files) and folders (storyboard spites and/or skin folders).
 
-The format is `{Beatmap number} {Artist} - {Song Title}`.
+The file name structure is `{Beatmap number} {Artist} - {Song Title}`.
 **Example:** [57950 SOUND HOLIC - Drive My Life](https://osu.ppy.sh/beatmapsets/57950)
 
 Please note that some very old beatmaps (for example, [Kenji Ninuma - DISCO PRINCE](https://osu.ppy.sh/beatmapsets/1) or [Dudelstudios - Angry Video Game Nerd Theme [MATURE CONTENT]](https://osu.ppy.sh/beatmapsets/66)), as well as unsubmitted beatmaps, do not follow the format.
 
-### Hidden Folders
+## Hidden folders
 
-#### Data
+These folders are hidden because any modifications to them could prevent osu! from starting correctly, or at all.
 
-**osu!data files.** Should not be tampered.
+### Data
 
-## Program files
+osu! data files. Contains some of osu!'s cache, like beatmap background cache and avatar caches. They should not be deleted, because they may be in use by osu!.
 
-### .db (Database file)
+## Files
 
-The .db files are beatmaps' data which only osu!.exe/osu!test.exe can utilise. Notepad can open the .db files but it will be mostly made up of encrypted characters.
+*Caution: Be careful with these files, you might break osu! if you are not careful.*
 
-**osu! database**
+## Database files (.db)
 
-- collection.db (Your "Collections" in-game. You can post it to forum for others to check out your "Collections". Actual beatmaps not provided.)
-- osu!.db (osu! beatmaps' cache. You know, that long start-up loading screen before you see Song Selection?)
-- presence.db (Cache of osu!players @ Chat Console)
-- scores.db (Stores historical Local scores)
+The database files are daabases that osu! requires to function propertly. The files contain vital information that osu! requires, such as saved scores, and the list of beatmaps saved on the player's device.
 
-### .cfg (Configuration files)
+- `collections.db`: Stores the player's "Collections" in-game. The player can post them to the forums for others to see. Actual beatmaps are not provided with the collection file.
```

Comment:
Since when has "posting" this file been a thing? I'd probably remove this.

Reply:
Remove the mention of collections.db, or the fact that you can post them online?